src/rgw/rgw_bucket.cc

```diff
@@ -149,17 +149,15 @@ void rgw_parse_url_bucket(const string &bucket, const string& auth_tenant,
  */
 int rgw_read_user_buckets(RGWRadosStore * store,
                           const rgw_user& user_id,
-                          RGWUserBuckets& buckets,
+                          rgw::sal::RGWBucketList& buckets,
```

Comment:
would try for a different namespace name here--maybe "storage?"

Reply:
I'm not sure I understand.  Are you objecting to sal?